README.md

```diff
@@ -99,12 +97,13 @@ ECP relies on the `certificate_config.json` file to read all the metadata inform
       "token_label": "YOUR_TOKEN_LABEL",
```

Comment:
Can you update this based on the currently supported fields? It should be label, slot, and module I believe.

Reply:
Good catch, thanks!